src/Toggle/MysqlActivatedToggleGateway.php

```diff
@@ -43,20 +62,21 @@ public function __construct( Connection $connection, IsToggleActive $toggleCheck
      */
     public function getAllMyActivatedToggles( $userIdentifier )
```

Comment:
Yeah, that should be removed, I think @VolodymyrSergeyev is already working on that, we came to a conclusion about this only this morning.